src/web/src/config.js

```diff
@@ -5,8 +5,18 @@ const rootUrl =
   process.env.NODE_ENV === 'production'
     ? urlBase
     : `http://localhost:${developmentPort}${urlBase}`;
-const apiBaseUrl = `${rootUrl}/api/v0`;
-const hubBaseUrl = `${rootUrl}/hub`;
+
+const apiBaseUrlFromLocalStorage = localStorage.getItem('apiBaseUrl');
```

Comment:
not related to the issue, but i found it easier to be able to override the API url so i don't have to run the entire backend locally just for a frontend change